spec/features/work_packages/select_work_package_row_spec.rb

```diff
@@ -29,21 +29,12 @@
 require 'spec_helper'
 require 'features/work_packages/work_packages_page'
 
-describe 'Select work package row', type: :feature do
+describe 'Select work package row', type: :feature, retry: 2, retry_wait: 5 do
   let(:user) { FactoryGirl.create(:admin) }
   let(:project) { FactoryGirl.create(:project) }
-  let(:work_package_1) {
-    FactoryGirl.create(:work_package,
-                       project: project)
-  }
-  let(:work_package_2) {
-    FactoryGirl.create(:work_package,
-                       project: project)
-  }
-  let(:work_package_3) {
-    FactoryGirl.create(:work_package,
-                       project: project)
-  }
+  let!(:work_package_1) { FactoryGirl.create(:work_package, project: project) }
+  let!(:work_package_2) { FactoryGirl.create(:work_package, project: project) }
+  let!(:work_package_3) { FactoryGirl.create(:work_package, project: project) }
```

Comment:
I have to admit that I really hate that pattern. The `!` is so hard to spot and it leaves you with no possibility to manipulate the behaviour in corner cases within the same file.